plugin/src/Config.lua

```diff
@@ -2,11 +2,23 @@ local strict = require(script.Parent.strict)
 
 local isDevBuild = script.Parent.Parent:FindFirstChild("ROJO_DEV_BUILD") ~= nil
 
+local Version = script.Parent.Parent.Version
+local realVersion = Version.Value:split(".")
+
+for i = 1, 3 do
+	local num = tonumber(realVersion[i])
+	if num then
+		realVersion[i] = num
+	else
+		error(("invalid version `%s` (field %d)"):format(realVersion[i], i))
+	end
+end
+
 return strict("Config", {
 	isDevBuild = isDevBuild,
 	codename = "Epiphany",
-	version = { 7, 3, 0 },
-	expectedServerVersionString = "7.2 or newer",
+	version = realVersion,
+	expectedServerVersionString = ("%d.%d or newer"):format(realVersion[1], realVersion[2]),
```

Comment:
It should be noted that the error message displayed to users when the `protocolVersion` is not compatible between Rojo server and plugin client will now display the latest known compatible server version, rather than the least compatible server version.

This shouldn't be an issue, but is worthwhile mentioning.

Reply:
My belief is that this is essentially the same thing, since if the major or minor version have been incremented, the server isn't compatible anymore on paper. It is worth noting though, thank you.